projects/fmcadc5/src/fmcadc5.c

```diff
@@ -364,49 +364,28 @@ int main(void)
 
 #ifdef IIO_SUPPORT
 	struct iio_device *adc_dev_0_desc;
-	struct iio_device *adc_dev_1_desc;
```

Comment:
Not sure I understand why axi-ad9625-1 is being removed. Is linux code also not up to date with current HDL ? 
https://github.com/analogdevicesinc/linux/blob/b39467c90e29766ecd6e08d3f6c3d7ff3c84b420/arch/microblaze/boot/dts/vc707_fmcadc5.dts#L101

Reply:
There are 2 spi chips and there is one instance for each (axi_ad9625_0_core and axi_ad9625_1_core) 
But only one is connected to to the axi_ad9625_dma controller ( https://github.com/analogdevicesinc/linux/blob/b39467c90e29766ecd6e08d3f6c3d7ff3c84b420/arch/microblaze/boot/dts/vc707_fmcadc5.dts#L69 ) to get data -> only one output channel buffer. 
See linux iio_info: [info_adc5.txt](https://github.com/analogdevicesinc/no-OS/files/7734142/info_adc5.txt)
On noos a similar link is done with the iio_axi_adc instance and if you check below the same ad9625_dmac was used for both